ci/test/00_setup_env_mac_functional.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
```

Comment:
Even thought it'll be unused, could add `NO_DEPENDS=1` here to make it clear that we aren't actually using depends.